unikernel.ml

```diff
@@ -1005,313 +1039,432 @@ struct
+                              ~user_policy unikernels_by_albatross_instance
+                              blocks_by_albatross_instance
+                              albatross_instance.configuration.name)
+                         ~icon:"/images/robur.png" ())
+                      ~header_list:[ ("X-MOLLY-CSRF", csrf) ]
+                      `OK
+                | None ->
+                    reply reqd ~content_type:"text/html"
+                      (Guest_layout.guest_layout
+                         ~page_title:"Resource policy error | Mollymawk"
+                         ~content:
+                           (Error_page.error_layout (missing_policy_error ()))
+                         ~icon:"/images/robur.png" ())
+                      `Internal_server_error)
+            | Error err ->
                 reply reqd ~content_type:"text/html"
                   (Guest_layout.guest_layout
                      ~page_title:"Resource policy error | Mollymawk"
                      ~content:
-                       (Error_page.error_layout (missing_policy_error ()))
+                       (Error_page.error_layout
+                          (missing_policy_error ~err:(Some err) ()))
                      ~icon:"/images/robur.png" ())
                   `Internal_server_error)
         | Error err ->
             reply reqd ~content_type:"text/html"
-              (Guest_layout.guest_layout
-                 ~page_title:"Resource policy error | Mollymawk"
-                 ~content:
-                   (Error_page.error_layout
-                      (missing_policy_error ~err:(Some err) ()))
+              (Guest_layout.guest_layout ~page_title:"500 | Mollymawk"
+                 ~content:(Error_page.error_layout err)
                  ~icon:"/images/robur.png" ())
               `Internal_server_error)
-    | Error err ->
+    | _ ->
         reply reqd ~content_type:"text/html"
           (Guest_layout.guest_layout ~page_title:"500 | Mollymawk"
-             ~content:(Error_page.error_layout err)
+             ~content:
+               (Error_page.error_layout
+                  {
+                    code = 500;
+                    success = false;
+                    title = "Albatross Instance Error";
+                    data =
+                      `String
+                        ("An error occured trying to find albatross instance "
+                        ^ Configuration.name_to_str instance_name);
+                  })
              ~icon:"/images/robur.png" ())
           `Internal_server_error
 
-  let unikernel_info albatross _ (user : User_model.user) reqd =
+  let unikernel_info stack albatross_instances _ (user : User_model.user) reqd =
     (* TODO use uuid in the future *)
-    Albatross.query albatross ~domain:user.name (`Unikernel_cmd `Unikernel_info)
+    Lwt_list.fold_left_s
+      (fun acc (_name, (pol : Albatross.t)) ->
+        match acc with
+        | Error _ as e -> Lwt.return e
```

Comment:
here we error out early -- so if there is an issue in any albatross configuration, the user only sees the error. instead, we should ignore the errors (or as discussed in #153 display "albatross A and B failed").

Reply:
this I propose to fix within this PR, and ignore the errors instead of bubbling them up.

At a later (#153) point in time, we can display appropriate errors.